### PR TITLE
configuration-settings

### PR DESCRIPTION
**Pull request by kghamilton89**
* New standalone Configuration Settings page.
* I modeled it after other Charts Settings pages, so there is an Overview and then the actual Settings section. Like the Shared Settings page, there is no Examples section.
* I kept the subsections defined on the Portals Settings page.
* I made the content changes that you suggested for the Shared Settings page, I'll go through and remove those settings which are enumerated here on that page.

### CODE REVIEW

**Comment by kghamilton89** [comment]
* Removed `expand-panels`, `show-panels`, and `periods` settings as they only apply to Time Chart.

**Comment by rodionos** [comment]
We also need to describe how width-units and height-units work when set at the group and widget level - they work differently from how it's described now in the [configuration] section.

**Comment by kghamilton89** [comment]
* I added an explanation for the `width-units` and `height-units` settings which describes the difference between specifying the setting in the `[configuration]` vs. `[widget]` or `[group]`.
* I ran all the example configurations through the plugin, no errors.

File: configuration/README.md [hunk 1]
```diff
@@ -1,54 +1,28 @@
...
+Name | Example | Description | &nbsp;
+:--|:--|:--|:--
+<a name="dialog-maximize"></a>[`dialog-maximize`](#dialog-maximize)  |  `dialog-maximize = true`  |  Expand dialog window to the entire page.<br>Open dialog window by clicking the widget header.<br>Possible values: `false`, `true`.<br>Default: `true`.  |  [↗](https://apps.axibase.com/chartlab/8bc68a84)
+<a name="scale"></a>[`scale`](#scale) | `scale = 0.8` | Chart scale.<br>Value must exceed `0.0`.<br>Default value: `1.0`.| [↗](https://apps.axibase.com/chartlab/1679114f)
+
+### Layout Settings
+
+Name | Example | Description | &nbsp;
+:--|:--|:--|:--
+<a name="height-units"></a>[`height-units`](#height-units) |`height-units = 2`| Number of widgets shown in the configuration vertical grid.<br>When specified in the `[configuration]` section, `height-units` determines the number of total widgets.<br>When specified in the `[widget]` or `[group]` section `height-units` determines the absolute widget height.<br>Default value: `4`.|[↗](https://apps.axibase.com/chartlab/645ea8fb)
```
Comment (rodionos) [inline configuration/README.md]: > Number of widgets shown in the configuration vertical grid.
Incorrect. See https://github.com/axibase/atsd/blob/master/portals/portal-settings.md#layout-settings. If the width-units in [conf] is 4 and width of the widget is set to 2, only 2 widgets will fit. However the description says this setting controls the NUMBER of widgets. There should be a precsiz

Comment (kghamilton89) [reply configuration/README.md]: Ah, I hadn't written with such a use case in mind. I was assuming that they would only declare one `height-unit` setting.

Comment (kghamilton89) [reply configuration/README.md]: I've made the settings more specific.

**Pull request merged**